src/main/resources/spy.properties

```diff
@@ -0,0 +1,2 @@
+modulelist=com.p6spy.engine.spy.P6SpyFactory
+useNanoTime=true
```

Comment:
Why do we want nanotime? I think it's not guaranteed better accuracy on most cpus/oss

Reply:
Many queries are sub millisecond and are displayed as just 1ms then.